scripts/_helpers.py

```diff
@@ -22,6 +23,33 @@
 
 REGION_COLS = ["geometry", "name", "x", "y", "country"]
 
+logger = logging.getLogger(__name__)
+
+
+def handle_exception(exc_type, exc_value, exc_traceback):
+    logger = logging.getLogger(__name__)
+    logger.setLevel(logging.INFO)
+    handler = logging.StreamHandler(stream=sys.stdout)
+    logger.addHandler(handler)
+
+    if issubclass(exc_type, KeyboardInterrupt):
+        sys.__excepthook__(exc_type, exc_value, exc_traceback)
+        return
+
+    tb = exc_traceback
+    while tb.tb_next:
+        tb = tb.tb_next
+    flname = tb.tb_frame.f_globals.get("__file__")
+    funcname = tb.tb_frame.f_code.co_name
+
+    logger.error(
+        "An error happened in module %r, function %r: %s",
+        flname,
+        funcname,
+        exc_value,
+        exc_info=(exc_type, exc_value, exc_traceback),
+    )
```

Comment:
This may be simplified with something like

``` python
    def handle_exception(exc_type, exc_value, tb):
      ...
      [Exclude Interrupt]
      ...
      logger.exception(
          "Exception {0}:{1} has occurred in the workflow.".format(exc_type, exc_value),
          exc_info=True, stack_info=True
      )
```

Reply:
Absolutely agree on keeping info on KeyBoardInterrupt. Added a functionality for that.